Resources/Prototypes/_SpaceCats/Actions/types.yml

```
@@ -0,0 +1,25 @@
+- type: entity
+  id: ToggleThermalVision
+  name: Switch Thermal vision
+  description: Switches Thermal vision
+  components:
+  - type: InstantAction
+    itemIconStyle: BigAction
+    priority: -20
+    icon:
+      sprite: _SpaceCats/Clothing/Eyes/Goggles/thermal.rsi
+      state: icon
+    event: !type:ToggleThermalVisionEvent
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)